skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/MacOsOpenGLRedrawer.kt

```diff
@@ -128,11 +135,25 @@ private open class AWTGLLayer(private val containerPtr: Long, setNeedsDisplayOnB
 
     open fun setNeedsDisplay() = setNeedsDisplayOnMainThread(ptr)
 
+    suspend fun display() = display.runAndAwait {
+        setNeedsDisplay()
+    }
+
     // Called in AppKit Thread
     protected open fun canDraw() = true
 
+    @Suppress("unused")
```

Comment:
Maybe comment that it's called from JNI?

Reply:
Done